.vscode/settings.json

```diff
@@ -1,10 +1,4 @@
 {
-    "dart.lineLength": 140,
-    "[dart]": {
-        "editor.rulers": [
-            140
-        ],
```

Comment:
my reasoning behind removing this is, if we want to leverage `dart format` in the future for our CI it will not honor this setting, so we'll be failing tests because of line length formatting issues. 

IMO for stuff like this i'd rather leverage the already written solutions instead of customizing our codebase, it will be one less thing we need to worry about.

Reply:
most of the changes in this PR are bumping lines to not break 140 characters, FYI.

Reply:
I've never seen a vscode settings file in our repo (not against the idea, just foreign to me). Do we need this file for CI? /Are we trying to support devs who use vscode having the same linting settings? I see you have `cmake.configureOnOpen: false` below which seems unrelated to dart.

Reply:
The reason it's included in our flutter sdk is because VS Code is the supported IDE for Flutter/Dart, so there is a lot of tooling for it built in, so it makes sense to include it in the project so we can share settings for formatting etc. across people working on it.

the cmake is unrelated to dart, im not familiar with that or what it does in this context.

another setting to call out is the `"editor.formatOnSave"` which will format files to dart standard upon save, its handy for not accidentally committing bad format and failing tests, i prefer to enforce this enabled on projects to avoid easy mistakes and wasted CI resources.

Reply:
That makes sense to me; thanks for the info!

Reply:
Strongly against this. I'd prefer to keep the 140 line length as it makes everything SO MUCH easier to read, doesn't result in terrible indentations, etc. We can update the `dart format` command to include `--line-length=140`

We should also update the makefile with a `format` command that calls the same function `dart format --line-length=140`